scipy/stats/tests/Vogelsang_trend_test.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct 24 15:34:53 2019
+@author: Solomon Vimal (solomon.vimal@gmail.com)
+Based on R code (from Prof. Rong Fu, Oct, 2019 AOS2019 Fall 2019, UCLA)
+& Gauss program by Vogelsang (https://msu.edu/~tjv/working.html) from 8/13/96
+"""
+
+from __future__ import division, print_function, absolute_import
+import sys
+import numpy as np
+import pandas as pd
+from numpy.linalg import inv
+
+def vogelsang_trend(y):
+    """
+    Computes the individual tests using the t-PS test of Vogelsang 
+    (1998, Econometrica)
+    
+    Test the null hypothesis that a time series with serial correlation has a 
+    slope with statistically significant trend.
+    
+    The test is evaluated at 90%, 95%, 97.5%, and 99.0% confidence levels.
+    
+    Parameters
+    ----------
+    y : array-like 
+        The time series for which the trend is to be calculated and tested for 
+        statistical significance.
+        
+    Returns
+    -------
+    slope: float
+        The slope of the trend function
+    pvalue: binary 
+        Binary result (1 or 0) for pass or fail for 4 levels of statistical 
+        significance at: 90%, 95%, 97.5%, and 99.0% confidence levels.
+        
+    See Also
+    --------
+    https://www.jstor.org/stable/2998543
+    
+    Notes
+    -----
+    This test is applicable when the time series has a serial auto correlation. 
+    which in mann-kendall and modified mann-kendall test will have a tendency 
+    for over rejection.
+    
+    This procedure computes the nonparametric estimator of the long run variance 
+    using Andrews (1991) AR(1) plug in optimal truncation lag.  
+    v is the vector of residuals and M is the trucation lag.  
+    M<0 uses the automatic bandwidth.  
+    Kernel is an integer representing the kernel that is used 1 = bartlett, 
```

Comment:
this is not actually implemented here, is it?

Reply:
No, this part is not implemented.